bats/tests/containers/docker-buildx-python3-uname.bats

```diff
@@ -0,0 +1,55 @@
+load '../helpers/load'
+
+local_setup() {
+    TEMP=/tmp
+    if is_windows; then
+        # We need to use a directory that exists on the Win32 filesystem
+        # so the ctrctl clients can correctly map the bind mounts.
+        # We can use host_path() on these paths because they will exist
+        # both here and in the rancher-desktop distro.
+        TEMP="$(win32env TEMP)"
+    fi
+    WORK_DIR="$TEMP/rd-bats-buildx-dockerDir"
+    RD_CONTAINER_ENGINE=moby
```

Comment:
We should not lock down `RD_CONTAINER_ENGINE` in a test (unless we are switching between engines).

We should instead define a helper and call this in every `@test` that does not apply to the selected engine:

```shell
skip_unless_using_docker() {
    if ! using_docker; then
        skip "This test only applied to the moby container engine"
    fi
}
```